src/ui/pages/Overview/Banners/Banners.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useRemoteConfigValue } from 'src/modules/remote-config/useRemoteConfigValue';
+import { MintBanner } from 'src/ui/DNA/components/MintBanner';
+import { UpgradeBanner } from 'src/ui/DNA/components/UpgradeBanner';
+import {
+  useAddressHasDnaUpgradeBackgroundPerk,
+  useShowDnaMintBanner,
+} from 'src/ui/DNA/shared/useShowDnaBanner';
+import { usePreferences } from 'src/ui/features/preferences';
+import { InviteFriendsBanner } from 'src/ui/features/referral-program/InviteFriendsBanner';
+import { Spacer } from 'src/ui/ui-kit/Spacer';
+import { ENABLE_DNA_BANNERS } from 'src/ui/DNA/components/DnaBanners';
+
+export function Banners({ address }: { address: string }) {
+  const {
+    data: referralProgramEnabled,
+    isLoading: isLoadingRemoteConfigValue,
+  } = useRemoteConfigValue('extension_referral_program');
+
+  const { preferences, setPreferences } = usePreferences();
+
+  const showDnaMintBanner = useShowDnaMintBanner(address);
+  const showUpgradeBanner = useAddressHasDnaUpgradeBackgroundPerk(address);
```

Comment:
I don't like that we make this requests despite statically knowing that `ENABLE_DNA_BANNERS` is `false`
We should refactor to bail out earlier. Maybe move out DNA banners into a separate component? In the same file

Reply:
Makes sense... Extracted `DnaBanners` component ✅